build.sbt

```
@@ -127,6 +127,15 @@ assembly / assemblyExcludedJars := {
   }
 }
 
+// protocol配下とルートのLICENSEが衝突してCIが落ちる
+// cf. https://github.com/sbt/sbt-assembly/issues/141
+assemblyMergeStrategy in assembly := {
+  case PathList(ps @ _*) if ps.last endsWith "LICENSE" => MergeStrategy.rename
+  case otherFile =>
+    val oldStrategy = (assemblyMergeStrategy in assembly).value
+    oldStrategy(otherFile)
+}
```

```suggestion
assemblyMergeStrategy in ThisBuild := {
  case PathList(ps @ _*) if ps.last endsWith "LICENSE" => MergeStrategy.rename
  case otherFile =>
    val oldStrategy = (assemblyMergeStrategy in ThisBuild).value
    oldStrategy(otherFile)
}
```

こうだったりしませんか？ (`Compile` スコープで落ちてるんで global scope 下で設定すると動くかも)